cypress/e2e/shared/flowsSidebarExportToClipboard.test.ts

```diff
@@ -0,0 +1,228 @@
+  return [
+    {
+      name: 'arduino',
+      token: `#define INFLUXDB_TOKEN "<INFLUX_TOKEN>"`,
+      org: `#define INFLUXDB_ORG "${org}"`,
+      bucket: `#define INFLUXDB_BUCKET "${bucket}"`,
+      query,
+      copyButtons: 3,
+    },
+    {
+      name: 'csharp',
+      token: `const string token = "${token}";`,
+      org: `const string org = "${org}";`,
+      bucket: `const string bucket = "${bucket}";`,
+      query,
+      copyButtons: 4,
+    },
+    {
+      name: 'go',
+      token: `const token = "${token}"`,
+      org: `const org = "${org}"`,
+      bucket: `const bucket = "${bucket}"`,
+      query,
+      copyButtons: 1,
+    },
+    {
+      name: 'java',
+      token: `String token = "${token}";`,
+      org: `String org = "${org}";`,
+      bucket: `String bucket = "${bucket}";`,
+      query,
+      copyButtons: 3,
+    },
+    {
+      name: 'javascript-node',
+      token: `const token = '${token}'`,
+      org: `const org = '${org}'`,
+      bucket: `const bucket = '${bucket}'`,
+      query,
+      copyButtons: 2,
+    },
+    {
+      name: 'kotlin',
+      token: `val token = "${token}"`,
+      org: `val org = "${org}"`,
+      bucket: `val bucket = "${bucket}"`,
+      query,
+      copyButtons: 3,
+    },
+    {
+      name: 'php',
+      token: `$token = '${token}';`,
+      org: `$org = '${org}';`,
+      bucket: `$bucket = '${bucket}';`,
+      query,
+      copyButtons: 2,
+    },
+    {
+      name: 'python',
+      token: `token = "${token}"`,
+      org: `org = "${org}"`,
+      bucket: `bucket = "${bucket}"`,
+      query,
+      copyButtons: 2,
+    },
+    {
+      name: 'ruby',
+      token: `token = '${token}'`,
+      org: `org = '${org}'`,
+      bucket: `bucket = '${bucket}'`,
+      query,
+      copyButtons: 2,
+    },
+    {
+      name: 'scala',
+      token: `val token = "${token}"`,
+      org: `val org = "${org}"`,
+      bucket: `val bucket = "${bucket}"`,
+      query,
+      copyButtons: 4,
+    },
+    {
+      name: 'swift',
+      token: `let token = "${token}"`,
+      org: `let org = "${org}"`,
+      bucket: `let bucket = "${bucket}"`,
+      query,
+      copyButtons: 2,
+    },
+  ]
+}
+
+describe('Flows', () => {
+  beforeEach(() => {
+    cy.flush()
+  })
+
+  describe('Flows Copy To Clipboard', () => {
+    beforeEach(() => {
+      cy.signin().then(() => {
+        cy.get('@org').then(({id}: Organization) => {
+          cy.fixture('routes').then(({orgs}) => {
+            cy.request('api/v2/authorizations').then(({body}) => {
+              cy.wrap(body.authorizations).as('tokens')
+            })
+            cy.visit(`${orgs}/${id}`)
+            cy.getByTestID('tree-nav')
+            cy.setFeatureFlags({
+              notebooks: true,
+              'flow-sidebar': true,
+              simpleTable: true,
+            })
+
+            cy.getByTestID('nav-item-flows').click()
+          })
+        })
+      })
+    })
+
+    it('Export to Clipboard as Code', () => {
```

Comment:
once a clipboard is exported as code, do we get redirected anywhere?

Reply:
no, we stay on the dialog box. 😿 honestly, i feel this entire thing was too excessive. we should have had this as 2 separate features. Copy as Code and View Code & Instructions.